deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,90 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+
+/* #C16E70 old rose #F2F3D9 beige #030027 blue*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
```

Comment:
Nova heeft hier geen flexbox van gemaakt. Ik denk omdat de items block-level elementen zijn die standaard onder elkaar staan. Probeer alleen een flexbox toe te voegen wanneer dit echt nodig is.